build.gradle

```diff
@@ -246,6 +246,7 @@ task fastbuild(dependsOn: [
 
 task travisbuild(dependsOn: [
     fastbuild,
+    ':dbfit-java:db2:integrationTest',
```

Comment:
The DB2 tests are ready to run as soon at the `before_script` section has run. The complete build is taking 7 minutes to complete. I don't think moving the order will change that.